types/src/storage.rs

```diff
@@ -219,6 +218,8 @@ pub struct PaginatedResult<T> {
     pub cursor: Option<(String, String)>,
 }
 
+type RawKeyValue = (Box<[u8]>, Box<[u8]>);
```

Comment:
Is there a reason why you uses Box<>? Can't we just use slices?

Reply:
Yes - this is the type that is returned from the rocksdb iterator: https://github.com/rust-rocksdb/rust-rocksdb/blob/3884a77bfa9f864f89bebf12e6cc34799fb51352/src/db_iterator.rs#L446

The difference between `Box<[u8]>` and `&[u8]` is that the Box  **owns** the bytes and can be converted into a String without copying.

Reply:
Got it thanks